src/controllers/user/user.controllers.js

```diff
@@ -15,20 +22,50 @@ const getUserDashboard = async (req, res) => {
   }
 }
 
+// const addVehicle = async (req, res) => {
+//   const {licenseNumber, vehicleName} = req.body;
+//   const _mail = req.user.email;
+//   try {
+//     const vehicle = await Vehicle.findVehicleByLicenseNumber(licenseNumber);
+//     if(vehicle != null) {
+//       res.redirect('/dashboard');
+//     }
+
+//     await Vehicle.addVehicle({
+//       userMail: _mail,
+//       licenseNumber: licenseNumber,
+//       vehicleName: vehicleName
+//     });
+//     res.redirect('/dashboard');
+//   } catch (err) {
+//     console.error(err);
+//   }
+// }
+
 const addVehicle = async (req, res) => {
-  const {licenseNumber, vehicleName} = req.body;
+  const { licenseNumber, vehicleName } = req.body;
   const _mail = req.user.email;
   try {
+    const vehicle = await Vehicle.findVehicleByLicenseNumber(licenseNumber);
+    if (vehicle != null) {
+      res.render('user/user.dashboard.ejs', {
+        error: 'Vehicle already exists', // Set an error message
```

Comment:
Does this way of flashing work?

Reply:
Co-pilot gave this to me, so I think it should work, but it is not being detected in our code, so I removed it